app/Controllers/feedController.php

```diff
@@ -530,10 +541,10 @@ public static function actualizeFeed(int $feed_id, string $feed_url, bool $force
 			}
 
 			$feedDAO->updateLastUpdate($feed->id(), false, $mtime);
-			$needFeedCacheRefresh |= ($feed->keepMaxUnread() != false);
+			$needFeedCacheRefresh |= ($feed->keepMaxUnread() !== false);
```

Comment:
Watch out, `!==` is indeed more strict, yes, but also less sure (more risk of errors because the tests are not lenient enough). This line is a good example.

```
<?php
$a = true;
$b = false;
$b |= $a;
echo json_encode($b), "\n";
// Output: 1
```

This is a PHP shortcoming, which changes type under the hood from `bool` to `int` (I hope this will be fixed in a future version), but it is anyway safer / more robust to test with *truthy* / *falsy*.

I am sending a patch with various fixes shortly